### DISCUSSION TITLE
Confused about Kilowatt hours from Embodied Carbon

### DISCUSSION THREAD

**Discussion by mgriffin-scottlogic** [cloud-carbon-footprint/cloud-carbon-footprint#1305] [Q&A] [answered]
I've been looking at the [`EmbodiedEmissionsEstimator`](https://github.com/cloud-carbon-footprint/cloud-carbon-footprint/blob/trunk/packages/core/src/embodiedEmissions/EmbodiedEmissionsEstimator.ts) and was a bit confused as to why it does a reverse calculation from CO2e to kWh. Isn't this creating energy that doesn't really exist on the cloud provider side?

I didn't know if it was an attempt at including the energy impact of manufacturing but my first thought was that the embodied estimator should just always return 0 for kilowattHours. That way I could feel more confident that the total kilowatt hours relate to the energy used by the cloud providers themselves. Would this be an acceptable change to make? It would also remove the last use of the estimateKwh function, meaning that could be removed too.

**Comment by ccasher** [comment, marked as answer]
Hi @mgriffin-scottlogic, thanks for bringing this up! Our initial thought when implementing this was since our methodology inherently just calculated the carbon impact and not the energy, we wanted to include the conversation to showcase the energy impact as well to get the overall picture for both carbon and energy estimations. 

We believe manufacturing impact can still be associated with cloud provider impact as a by-product of the cloud usage. However, if we see this is something that CCF users would want, we could consider adding a feature to disable it!

**Reply by mgriffin-scottlogic** [reply]
Ok, so you would rather there was a common config option say, that would disable embodied energy estimation? It feels like if the energy calculation is intentional then it should be more clear - possibly even specifying a region of manufacture, as that's where the carbon intensity used in the reverse calculation should be sourced from.

**Reply by ccasher** [reply]
Yes, we are being intentional about providing the energy estimation, which is why we did not originally consider the option to disable the calculation. Currently in the embodied emissions estimator, we pass through the usage region, and if that is not available, we default to the average emissions factor we have in our constants file. This approach seemed to make sense to us, but if you feel this may be too big of an assumption or if you have any suggestions on where to grab information about regions where processors are manufactured, this would be an awesome discussion to have.

In terms of being more clear, would it be helpful if we explicitly called this out in our documentation? Open to any suggestions on how to approach this.
